src/ctapipe/reco/impact.py

```diff
@@ -100,12 +100,6 @@ class ImPACTReconstructor(HillasGeometryReconstructor):
 
     """
 
-    minimiser = traits.CaselessStrEnum(
```

Comment:
So this traitlets wasn't used anymore anyway? Just the config was still there?

Reply:
Yep...